taxonomy/api/v1/serializers.py

```diff
@@ -0,0 +1,61 @@
+"""
+Taxonomy API serializers.
+"""
+from rest_framework.serializers import ModelSerializer
+
+from taxonomy.models import CourseSkills, Job, JobPostings, JobSkills, Skill
+
+
+class JobSerializer(ModelSerializer):
+
+    class Meta:
+        model = Job
+        fields = '__all__'
+
+
+class SkillSerializer(ModelSerializer):
```

Comment:
Just wanted to bring this into notice that `SkillSerializer` already exists here https://github.com/edx/taxonomy-connector/blob/master/taxonomy/serializers.py#L10

Reply:
That serializer was not API specific, That's why I created this one, New all-new serializers are for API views.